addons/interface/XEH_postInit.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 
-if !(isClass(configFile >> "CfgPatches" >> "ace_interaction")) then {
+if (!isClass (configFile >> "CfgPatches" >> "ace_interaction")) then {
```

Comment:
I know this is standard in ACE and other project, still not sure how I feel about it though